packages/console-app2/src/main.tsx

```diff
@@ -3,17 +3,15 @@
 //
 
 import React from 'react';
-
-import JSONTree from '@dxos/react-json-tree'
-
 import { Dimmer, Loader, Grid, Message } from 'semantic-ui-react';
```

Comment:
Change to material-ui

Reply:
Same as above, I don't understand how this is related to material-ui

Reply:
Same story, material UI to come, in the next PRs. Here it's clear the import was merely not needed and @vitaliyworks removed the code.